### ISSUE TITLE
why we cannot reproduce the result of logit-adjusted loss

### ISSUE THREAD

**Issue by zhuhongtiandi** [Chumsy0725/logit-adj-pytorch#8]
In the README.md, the result of logit-adjusted loss is 0.7857, but the result of our reproduction with the same code and data is 0.7421. Do we miss anything? On the other hand, the reproduction result of post-hoc logit-adjustment is 0.8062, which looks pretty good.

**Comment by zhuhongtiandi** [comment]
> In the README.md, the result of logit-adjusted loss is 0.7857, but the result of our reproduction with the same code and data is 0.7421. Do we miss anything? On the other hand, the reproduction result of post-hoc logit-adjustment is 0.8062, which looks pretty good.

We have found the reason of 0.8062, which is that we use a better pre-trained model instead of ERM. But the issue about 0.7421 is still open.

**Comment by Xingcun-Li** [comment]
Is this problem solved?
